dist/api_data.py

```diff
@@ -1605,6 +1605,10 @@ def __ge__(self, other):
 ]),
 
 'WT_SESSION.begin_transaction' : Method([
+    Config('evict_skip', 'false', r'''
```

Comment:
Do we want these options to be `undoc=True`? This is option has sharp edges, so we might not want the typical user (if there is such a beast) using it.

Reply:
Yes, I will make them all undoc.